fuel-tx/src/transaction/validity.rs

```diff
@@ -112,6 +112,8 @@ impl Input {
                     recover_address()?
                 };
 
+                // This error is reached during fuzzing often, verify it is safe to disable it
+                #[cfg(not(any(fuzzing, feature = "test-helpers")))]
```

Comment:
I'm curious what problem you had here and how to reproduce it? Maybe we will have some ideas on how to address that in a better way

Reply:
I don't have one at hand anymore, but removing the line and running the fuzzer should uncover it quickly. We can simply remove that for now and re-add it later on when you have CI fuzzing.